tensorflow-cc-toolfile.spec

```diff
@@ -0,0 +1,22 @@
+### RPM external tensorflow-cc-toolfile 1.0
+Requires: tensorflow-cc tensorflow-c-toolfile
+%prep
+
+%build
+
+%install
+
+mkdir -p %i/etc/scram.d
+cat << \EOF_TOOLFILE >%i/etc/scram.d/tensorflow-cc.xml
+<tool name="tensorflow-cc" version="@TOOL_VERSION@">
+  <lib name="tensorflow_cc"/>
+  <client>
+    <environment name="TENSORFLOW_CC_BASE" default="@TOOL_ROOT@"/>
+    <environment name="LIBDIR" default="$TENSORFLOW_CC_BASE/lib:$TENSORFLOW_C_BASE/lib"/>
+    <environment name="INCLUDE" default="$TENSORFLOW_CC_BASE/include:$TENSORFLOW_C_BASE/include"/>
```

Comment:
tensorflow-c-toolfile is already setting 
 - $TENSORFLOW_C_BASE/lib
 - $TENSORFLOW_C_BASE/include
so you do not need to set them here. all you need to do is to add dependency on tensorflow-c i.e. cleanup the above two lines and add ```<use name="tensorflow-c"/>```